packages/vitest/src/node/plugins/utils.ts

```diff
@@ -24,8 +24,6 @@ export function resolveOptimizerConfig(_testOptions: DepsOptimizationOptions | u
     }
   }
   else {
-    const root = testConfig.root ?? process.cwd()
-    const cacheDir = testConfig.cache !== false ? testConfig.cache?.dir : undefined
```

Comment:
We still need to handle it during the deprecation period

Reply:
I've updated it. I'm new to the optimizer config, not sure if this is right change, could you please help to review again? Thank you so much!

Reply:
I would expect `test.cache.dir` to just set `cacheDir`, everything else should just refer to `config.cacheDir`